TaskManager/Home/Model/ModelTask.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ModelTask.swift
+//  TaskManager
+//
+//  Created by user on 15.09.2022.
+//
+
+import Foundation
+
+struct ModelTask: Hashable {
+    var task: String
+    
+    
+    
+    static func testDataActive() -> [ModelTask] {
```

Comment:
Let's move this methods to a separate class DataStorageImp. Also no need to make it static let's have a protocol DataStorage and access this via delegate.